app/display.h

```diff
@@ -36,57 +37,76 @@
     prints(0, 30, str)
 
 #define display_cpu_clk(freq) \
-    printf(1, 10, "%i MHz", freq)
+    printf(1, 10, "%iMHz", freq)
 
 #define display_cpu_addr_mode(str) \
-    prints(1, 20, str)
+    prints(5, 76, str)
 
 #define display_l1_cache_size(size) \
-    printf(2, 10, "%kB", (uintptr_t)(size))
+    printf(2, 9, "%6kB", (uintptr_t)(size));
 
 #define display_l2_cache_size(size) \
-    printf(3, 10, "%kB", (uintptr_t)(size))
+    printf(3, 9, "%6kB", (uintptr_t)(size));
 
 #define display_l3_cache_size(size) \
-    printf(4, 10, "%kB", (uintptr_t)(size))
+    printf(4, 9, "%6kB", (uintptr_t)(size));
+
+#define display_memory_size(size) \
+    printf(5, 9, "%6kB", (uintptr_t)(size));
 
 #define display_l1_cache_speed(size) \
-    printf(2, 19, "%kB/s", (uintptr_t)(size))
+    printf(2, 18, "%S6kB/s", (uintptr_t)(size))
 
 #define display_l2_cache_speed(size) \
-    printf(3, 19, "%kB/s", (uintptr_t)(size))
+    printf(3, 18, "%S6kB/s", (uintptr_t)(size))
 
 #define display_l3_cache_speed(size) \
-    printf(4, 19, "%kB/s", (uintptr_t)(size))
+    printf(4, 18, "%S6kB/s", (uintptr_t)(size))
 
 #define display_ram_speed(size) \
-    printf(5, 19, "%kB/s", (uintptr_t)(size))
+    printf(5, 18, "%S6kB/s", (uintptr_t)(size))
 
-#define display_memory_size(size) \
-    printf(5, 10, "%kB", (uintptr_t)(size))
+#define display_status(status) \
+    prints(7, 68, status)
 
-#define display_dmi_mb(sys_ma, sys_sku) \
-    dmicol = prints(23, dmicol, sys_man); \
-    prints(23, dmicol + 1, sys_sku);
+#define display_threading(nb, mode) \
+    printf(7,31, "%uT (%s)", nb, mode);
+
+#define display_threading_disabled() \
+    prints(7,31, "Disabled");
+
+#define display_cpu_topo_hybrid(nb) \
+    printf(7, 5, "%u Threads (Hybrid)", nb);
 
-#define display_available_cpus(count) \
-    printi(7, 10, count, 4, false, false)
+#define display_cpu_topo_multi_socket(nbs, nbc, nbt) \
```

Comment:
An example of over-abbreviation. Would be easier to read with "(num_sockets, num_cores, num_threads)"